src/features/HelpAndKnowledge/SolutionsHelp/SolutionDetails/Solutions/Generic/_components/ContractorForm/index.tsx

```diff
@@ -24,7 +24,7 @@ import dirtyInput from 'utils/formUtil';
 import { ModeType } from '../ContractorModal';
 
 type FormValues = {
-  contractorTitle: string | null;
+  contractorTitle: string;
   contractorName: string;
 };
```

Comment:
Was there a specific reason to remove `null`? 

After looking more closely we actually should correlate this type directly to the query.  This is more true to the data and will prevent future bugs

```
type FormValues = {
  contractorTitle: SolutionContractorType['contractorTitle'];
  contractorName: SolutionContractorType['contractorName'];
};
```

Reply:
I was gonna say the null come from query lol. Perfect i will update it!